core/config/locales/en.yml

```diff
@@ -1247,6 +1247,8 @@ en:
       done: DONE
       default: DEFAULT
       filter_by: FILTER BY
+      max_price: 'Max'
+      min_price: 'Min'
```

Comment:
@mad-eel let's add `min` and `mix` keys in the main scope so we can later re-use it